API/Services/PhotoService.cs

```diff
@@ -22,8 +25,27 @@ public PhotoService(IOptions<CloudinarySettings> config)
         );
 
         _cloudinary = new Cloudinary(acc);
```

Comment:
Possible to register the Cloudinary in Startup.cs and then reject it into the PhotoService?